From dfbdd324769f542b6b89499c1b7f0f4322100bc6 Mon Sep 17 00:00:00 2001
From: wfaulk <wfaulk@users.noreply.github.com>
Date: Wed, 7 Aug 2019 18:57:43 -0400
Subject: [PATCH] fix utf8 support

in oping.c:923 in has_utf8() the #if HAVE_NCURSESW_NCURSES_H is
wrong. in consequence of this hist_symbols_utf8 is not used and the
the graph type prettyping is not realy pretty. the name of the header
file is curses.h see also oping.c:85 or the third changelog entry from
1997/05/31

https://github.com/octo/liboping/issues/36

[Upstream: https://github.com/octo/liboping/pull/49/commits/a313923ec12d11daa8e70497c6b7b5d50965821b]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 src/oping.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/oping.c b/src/oping.c
index 87ca81a..62dbb67 100644
--- a/src/oping.c
+++ b/src/oping.c
@@ -903,7 +903,7 @@ static void time_calc (struct timespec *ts_dest, /* {{{ */
 #if USE_NCURSES
 static _Bool has_utf8() /* {{{ */
 {
-# if HAVE_NCURSESW_NCURSES_H
+# if HAVE_NCURSESW_CURSES_H
 	if (!opt_utf8)
 	{
 		/* Automatically determine */
-- 
2.25.1

